src/components/Links/ExternalLink/ExternalLink.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Link as DSLink,
+  type LinkTypes,
+} from "@nypl/design-system-react-components"
+import { type ReactNode } from "react"
+
+interface ExternalLinkProps {
+  href: string
+  children: ReactNode
+  className?: string
+  target?: "_blank" | "_parent" | "_self" | "_top"
+  type?: LinkTypes
+  fontSize?: string | Record<string, string>
+  fontWeight?: string
+  display?: string
+  rel?: string
+  mb?: string
+  mt?: string
+  lang?: string
+  isUnderlined?: boolean
+  disabled?: boolean
+}
+
+// TODO: once 2ad is phased out, replace with DS v3 Link which can wrap a
+// next link.
+
+/**
+ * The ExternalLink component is a utility that renders a Design System link with some defaults
+ * set for the Research Catalog to avoid duplication.
+ */
+const ExternalLink = ({
```

Comment:
This and `RCLink` share a lot of code that I'm afraid eventually it will become difficult to maintain both. Really, the only big difference is how `RCLink` handles the href and the state for the subnav (active and white focus ring). It's okay for now but if it becomes harder to maintain, or if we forget to use these and use a different link, then we should revisit this.

Reply:
I agree that they share a lot of functionality right now, but I think that will change when we change RCLink to wrap the link in the default Next Link again.

Reply:
It's possible that they can be combined into a single component if we add props to differentiate them, however.